Makefile

```diff
@@ -0,0 +1,5 @@
+default: build
+build:
+	docker build -t ustwo/wordy-slack-bot .
+run:
+	docker run --env SLACK_TOKEN=$(SLACK_TOKEN) -d ustwo/wordy-slack-bot
```

Comment:
You can put a name on it:

```sh
docker run --env SLACK_TOKEN=$(SLACK_TOKEN) --name wordy-bot -d ustwo/wordy-slack-bot
```

Then you can create a make task to remove the container:

```sh
rm:
  docker rm -vf wordy-bot
```